vectordatasource/transform.py

```diff
@@ -8906,6 +8927,132 @@ def remap_viewpoint_kinds(shape, props, fid, zoom):
     return (shape, props, fid)
 
 
+def _list_of_countries(value):
+    """
+    Parses a comma or semicolon delimited list of ISO 3166-1 alpha-2 codes,
+    discarding those which don't match our expected format.
+
+    Returns a list of lower-case, stripped country codes.
+    """
+
+    from re import match
+    from re import split
+
+    countries = []
+    candidates = split('[,;]', value)
+
+    for candidate in candidates:
+        # should have an ISO 3166-1 alpha-2 code, so should be 2 ASCII
+        # latin characters.
+        candidate = candidate.strip().lower()
+        if match('[a-z][a-z]', candidate):
+            countries.append(candidate)
+
+    return countries
+
+
+def unpack_viewpoint_claims(shape, props, fid, zoom):
+    """
+    Unpack OSM "claimed_by" list into viewpoint kinds.
+
+    For example; "claimed_by=AA,BB,CC" should become "kind:aa=country,
+    kind:bb=country, kind:cc=country" (or region, etc... as appropriate for
+    the main kind, which should be "unrecognized_TYPE".
+    """
+
+    prefix = 'unrecognized_'
+    kind = props.get('kind')
+    claimed_by = props.get('claimed_by')
```

Comment:
We don't want to export `claimed_by` in the properties, only adding new `kind:*` properties.

Where do we drop `claimed_by` property?

Reply:
Ah, we don't drop it - yet! Fixed in b7807eef.